tests/integration/send-tokens/send-tokens.spec.ts

```diff
@@ -54,12 +56,87 @@ describe(`Send tokens flow`, () => {
       expect(errorMsg).toBeFalsy();
     });
 
+    it('validates that the address used is from different network', async () => {
+      await sendForm.inputToAmountField('0.000001');
+      await sendForm.inputToAddressField('STRE7HABZGQ204G3VQAKMDMVBBD8A8CYKET9M0T');
+      await sendForm.clickPreviewTxBtn();
+      const errorMsgElement = await sendForm.page.$$(sendForm.getSelector('$stxAddressFieldError'));
+      const errorMessage = await errorMsgElement[0].innerText();
+      expect(errorMessage).toContain('The address is for the incorrect Stacks network');
+    });
+
     it('validates against a negative amount of tokens', async () => {
       await sendForm.inputToAmountField('-9999');
       await sendForm.inputToAddressField('ess-pee');
       await sendForm.clickPreviewTxBtn();
       const errorMsg = await sendForm.page.isVisible(sendForm.getSelector('$amountFieldError'));
       expect(errorMsg).toBeTruthy();
     });
+
+    it('validates that token amount has more than 6 decimal places', async () => {
+      await sendForm.inputToAmountField('0.0000001');
+      await sendForm.inputToAddressField('SP15DFMYE5JDDKRMAZSC6947TCERK36JM4KD5VKZD');
+      await sendForm.clickPreviewTxBtn();
+      const errorMsgElement = await sendForm.page.$$(sendForm.getSelector('$amountFieldError'));
+      const errorMessage = await errorMsgElement[0].innerText();
+      expect(errorMessage).toEqual('STX can only have 6 decimals');
+    });
+
+    it('validates that token amount is greater than the available balance', async () => {
+      await sendForm.inputToAmountField('999999999');
+      await sendForm.inputToAddressField('SP15DFMYE5JDDKRMAZSC6947TCERK36JM4KD5VKZD');
+      await sendForm.clickPreviewTxBtn();
+      const errorMsgElement = await sendForm.page.$$(sendForm.getSelector('$amountFieldError'));
+      const errorMessage = await errorMsgElement[0].innerText();
+      expect(errorMessage).toContain('Insufficient balance');
+    });
+  });
+});
+
+describe('Preview for sending token', () => {
+
+  let browser: BrowserDriver;
+  let walletPage: WalletPage;
+  let sendForm: SendPage;
+
+  beforeEach(async () => {
+    browser = await setupBrowser();
+    walletPage = await WalletPage.init(browser, ScreenPaths.INSTALLED);
+    await walletPage.signIn(SECRET_KEY_2);
+    await walletPage.waitForHomePage();
+    await walletPage.goToSendForm();
+    sendForm = new SendPage(walletPage.page);
+  }, 30_000);
+
+  afterEach(async () => {
+    try {
+      await browser.context.close();
+    } catch (error) {
+    }
+  });
+
+  it('expect that the preview is shown', async () => {
```

Comment:
I think these statements might read better like, `it('should show the preview',...`

Same for one after, `it('should show the preview when there...`